src/components/DatasetCard/index.tsx

```diff
@@ -66,14 +66,14 @@ const DatasetCard: React.FunctionComponent<DatasetCardProps> = ({
             />
             {/* Spacer div to push `Descriptions` to the bottom of the card */}
             <div className={styles.spacer}/>
-            <Descriptions column={1} size="small">
-                {userData.totalCells && <Descriptions.Item label="Number of Cells">
+            <Descriptions column={1} size="small" colon={false}>
+                {userData.totalCells && <Descriptions.Item label="Number of Cells:">
```

Comment:
Makes colons part of the label to prevent a weird alignment bug with multiline labels